radar/radar.go

```diff
@@ -77,7 +90,10 @@ func (radar *Radar) Scanner(logger lager.Logger, resourceName string) ifrit.Runn
 			case <-signals:
 				return nil
 
-			case <-ticker.C:
+			case newInterval := <-radar.updateIntervalChan:
```

Comment:
@vito paired with your comment above I actually think the channel is going to work out nicely.  I am going to make it so that a brand new timer with the correct interval is sent down the channel instead.

I like this approach a lot better than making scan return an interval when it doesn't really have a reason to.